sway/sway-input.5.scd

```diff
@@ -239,6 +239,25 @@ correct seat.
 	by default) for the seat. This is primarily useful for video games. The
 	"escape" command can be used at runtime to escape from a captured client.
 
+*seat* <name> shortcuts_inhibitor enable|disable|active|inactive|toggle
```

Comment:
Maybe we should rename `active` to `activate`, just to make it clear the command performs an action and isn't read-only.

Reply:
@RedSoxFan suggested the use of `parse_boolean()` which handles the special `toggle` case and accepts `yes`, `on`, `true`, `enable`, `enabled`, and `active`. Neither of those matches particularly well and I also don't like the fact that it silently accepts anything else as false. So I would eliminate it again and explicitly match for `activate`, `deactivate` and `toggle` as before, perhaps without the enum boilerplate. Would that be acceptable? Alternatively, we could extend `parse_boolean()`...

Reply:
Switched from parse_boolean back to explicit parsing. Also simplified the control flow. I could comfortably live with this latest incarnation, fwiw.